### PR TITLE
January news

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: content/blog/2024-01-26-january-news/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,622 @@
...
+<!-- Before sending DELETE THE INDEX_CACHE and re-knit! -->
+
+Dear rOpenSci friends, it's time for our monthly news roundup!
+<!-- blabla -->
+You can read this post [on our blog](`r url`).
+Now let's dive into the activity at and around rOpenSci!
+
+## rOpenSci HQ
+
+### R-universe now builds MacOS ARM64 binaries for use on Apple Silicon (aka M1/M2/M3) systems.
```
Comment (steffilazerte) [inline content/blog/2024-01-26-january-news/index.Rmd:46]: Perhaps split to make the section look a bit nicer
```suggestion
### R-Universe
The R-Universe now builds MacOS ARM64 binaries for use on Apple Silicon (aka M1/M2/M3) systems!
```

File: content/blog/2024-01-26-january-news/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,622 @@
...
+Hosted by Steffi LaZerte and various community hosts. 
+Everyone welcome. 
+No RSVP needed. 
+Consult our [Events](/events) page to find your local time and how to join.
+
+- [Tuesday, February 6th, 9:00 Americas Pacific (17:00 UTC)](/events/coworking-2024-02/), R-Universe Office Hours. With cohost [Jeroen Ooms](/author/jeroen-ooms/) and [Steffi LaZerte](/author/steffi-lazerte/).
+    - Explore what the [R-Universe](https://r-universe.dev/) has to offer;
+    - Create your own R-Universe;
+    - Ask questions or troubleshoot your R-Universe problems with the cohost and other attendees.
+
```
Comment (steffilazerte) [inline content/blog/2024-01-26-january-news/index.Rmd:64]: I put Alex in for now, but if anyone else chooses to join, we can change that later? Or should I just leave my other host empty for now?

```suggestion

- [Tuesday, March 5th, 9:00 Australia Western (1:00 UTC)](/events/coworking-2024-03/), Dates, Times and Timezones in R. With cohost [Steffi LaZerte](/author/steff-lazerte/) and [Alex Koiter](/author/alex-koiter/).
    - Explore resources for working with dates, times, and timezones in R
    - Work on a project dealing with dates and times
    - Ask questions or troubleshoot your timezone problems with the cohost and other attendees.
    
```

File: content/blog/2024-01-26-january-news/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,622 @@
...
+Some useful tips for R package developers. :eyes:
+
+### How to get feedback from package users
+
+Matthias Grenié recently asked an intriguing question on [Mastodon](https://ecoevo.social/@LeNematode/111731312906546444): "Do you know any good resource to collect feedback from users? Or general frameworks?". Among the answers, apart from [rOpenSci software peer-review](https://digitalcourage.social/@rstub/111736172101121740) by Ralf Stubner, we note the mention of [friction logs](https://hachyderm.io/@zkamvar/111732172001573387) by Zhian N. Kamvar and of a [dedicated feedback channel](https://fosstodon.org/@almenal99/111733782615742712) by Alvaro Mendoza.
+
+### A tip when refactoring a test file
+
+When refactoring a test file used with testthat (for instance removing top-level code in favor of [test helpers](https://r-pkgs.org/testing-design.html#sec-tests-files-overview)), you can add a [`return()`](https://mastodon.social/@gaborcsardi@fosstodon.org/111719992256767987) in the middle of the file so that when you test the file, only the tests above the return statement are run.
```
Comment (steffilazerte) [inline content/blog/2024-01-26-january-news/index.Rmd:598]: So cool!

File: content/blog/2024-01-26-january-news/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,622 @@
...
+This way, you can make your way through your script, refactoring it little by little and moving the return statement towards the bottom as you go.
+
+### Code clubs
+
+Heidi Seibold wrote about [Code clubs](https://heidiseibold.ck.page/posts/code-clubs-good-coding-practices-for-all) in her newsletter.
+A great post to find out about them!
+She even mentions [rOpenSci coworking sessions](/coworking).
```
Comment (steffilazerte) [inline content/blog/2024-01-26-january-news/index.Rmd:605]: :tada:

File: content/blog/2024-01-26-january-news/index.Rmd [hunk 5]
```diff
@@ -0,0 +1,622 @@
...
+
+### Cold-blooded software
+
+Also not R-specific, but have you heard of the phrase ["cold-blooded software"](https://dubroy.com/blog/cold-blooded-software/)?
```
Comment (steffilazerte) [inline content/blog/2024-01-26-january-news/index.Rmd:609]: Hah! How great!

File: content/blog/2024-01-26-january-news/index.md [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+
+## Software Peer Review
+
+There are seventeen recently closed and active submissions and 4 submissions on hold. Issues are at different stages: 
+
+* Two at ['6/approved'](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A6/approved):
+
+     * [comtradr](https://github.com/ropensci/software-review/issues/613), Interface with the United Nations Comtrade API. Submitted by [paulbochtler](https://github.com/datapumpernickel).
```
Comment (steffilazerte) [inline content/blog/2024-01-26-january-news/index.md:71]: Should this be in the New packages section above?

File: content/blog/2024-01-26-january-news/index.md [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+
+* [Participación  Latinoamérica en el Programa de Campeonas y Campeones de rOpenSci](/es/blog/2024/01/09/champions-program-latinr) by Yanina Bellini Saibene, Elio Campitelli, Marcos Prunello, Mauro Lepore, Haydee Svab, Beatriz Milz, Carolina Pradier, Cesar Luis Aybar Camacho, and Pao Corrales. Detalles de la participación latinoamericana en el primer año del programa de campeonas y campeones de rOpenSci.
+
+* [rOpenSci 2023 Code of Conduct Transparency Report](/blog/2024/01/12/transparency2023) by Yanina Bellini Saibene, Mark Padgham, and Kara Woo. rOpenSci 2023 Code of Conduct Transparency Report.
+
+
+
+### Tech Notes
+
+* [R-universe now builds MacOS ARM64 binaries for use on Apple Silicon (aka M1/M2/M3) systems](/blog/2024/01/14/runiverse-arm64) by Jeroen Ooms. R-universe now builds MacOS ARM64 binaries for use on Apple Silicon (aka M1/M2/M3) systems.
```
Comment (steffilazerte) [inline content/blog/2024-01-26-january-news/index.md:131]:
```suggestion
* [R-universe now builds MacOS ARM64 binaries for use on Apple Silicon (aka M1/M2/M3) systems](/blog/2024/01/14/runiverse-arm64) by Jeroen Ooms.
```

File: content/blog/2024-01-26-january-news/index.md [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+## Software :package:
+
+### New packages
+
+
+
+
+The following  package recently became a part of our software suite:
+
++ [GLMMcosinor](https://docs.ropensci.org/GLMMcosinor), developed by Rex Parsons together with Oliver Jayasinghe, Nicole White, and Oliver Rawashdeh: Allows users to fit a cosinor model using the glmmTMB framework. This extends on existing cosinor modelling packages, including cosinor and circacompare, by including a wide range of available link functions and the capability to fit mixed models. The cosinor model is described by Cornelissen (2014) <doi:10.1186/1742-4682-11-16>.  It is available on [CRAN]( https://CRAN.R-project.org/package=GLMMcosinor). It has been [reviewed](https://github.com/ropensci/software-review/issues/603) by Michael Sachs and Joaquin Cavieres.
```
Comment (steffilazerte) [inline content/blog/2024-01-26-january-news/index.md:55]: No sure about the DOI here... does that work? Maybe try the link?
```suggestion
+ [GLMMcosinor](https://docs.ropensci.org/GLMMcosinor), developed by Rex Parsons together with Oliver Jayasinghe, Nicole White, and Oliver Rawashdeh: Allows users to fit a cosinor model using the glmmTMB framework. This extends on existing cosinor modelling packages, including cosinor and circacompare, by including a wide range of available link functions and the capability to fit mixed models. The cosinor model is described by Cornelissen (2014) <https://doi.org/10.1186/1742-4682-11-16>.  It is available on [CRAN]( https://CRAN.R-project.org/package=GLMMcosinor). It has been [reviewed](https://github.com/ropensci/software-review/issues/603) by Michael Sachs and Joaquin Cavieres.
```

File: content/blog/2024-01-26-january-news/index.Rmd [hunk 9]
```diff
@@ -0,0 +1,622 @@
...
+}
+```
+
+## Call for maintainers
+
+If you're interested in maintaining any of the R packages below, you might enjoy reading our blog post [What Does It Mean to Maintain a Package?](/blog/2023/02/07/what-does-it-mean-to-maintain-a-package/).
+
+[gendercoder](https://docs.ropensci.org/gendercoder/), Provides functions and dictionaries for recoding of freetext gender responses into more consistent categories. [Issue for volunteering](https://github.com/ropensci/gendercoder/issues/56).
+
+[sofa](https://docs.ropensci.org/sofa/), Provides an interface to the NoSQL database [CouchDB](http://couchdb.apache.org). [Issue for volunteering](https://github.com/ropensci/sofa/issues/82)
```
Comment (steffilazerte) [inline content/blog/2024-01-26-january-news/index.Rmd:582]:
```suggestion
[sofa](https://docs.ropensci.org/sofa/), Provides an interface to the NoSQL database [CouchDB](https://couchdb.apache.org). [Issue for volunteering](https://github.com/ropensci/sofa/issues/82)
```

**Review by steffilazerte — COMMENTED** [review]
Looks good! I added in the coworking :grin:

* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md

File: content/blog/2024-01-26-january-news/index.Rmd (hunk 2 above)
Comment (maelle) [reply content/blog/2024-01-26-january-news/index.Rmd:65]: sounds good!

File: content/blog/2024-01-26-january-news/index.Rmd (hunk 3 above)
Comment (maelle) [reply content/blog/2024-01-26-january-news/index.Rmd:604]: Right?!

File: content/blog/2024-01-26-january-news/index.md (hunk 6 above)
Comment (maelle) [reply content/blog/2024-01-26-january-news/index.md:79]: good catch! i see it was reviewed a second time, will add it "manually".

**Pull request merged**